atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/ThreadLocalWrappingTransactionManager.java

```diff
@@ -1,41 +0,0 @@
-/**
- * Copyright 2015 Palantir Technologies
- *
- * Licensed under the BSD-3 License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://opensource.org/licenses/BSD-3-Clause
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.atlasdb.transaction.impl;
-
-import com.palantir.atlasdb.transaction.api.LockAwareTransactionManager;
-import com.palantir.atlasdb.transaction.api.Transaction;
-import com.palantir.atlasdb.transaction.api.TransactionWrapper;
-import com.palantir.common.concurrent.ExecutorInheritableThreadLocal;
-
-public class ThreadLocalWrappingTransactionManager extends WrappingTransactionManager {
```

Comment:
This brings up something interesting where it looks like internally we wrap with several additional transaction managers including:

* tracing (soon open source)
* `com.palantir.atlasdb.transaction.impl.CachingTransactionManager`
* `com.palantir.atlasdb.transaction.impl.ThreadLocalWrappingTransactionManager`

But for current OS Atlas we don't wrap with these transaction managers. Not sure if we want to hook these up by default in `com.palantir.atlasdb.factory.TransactionManagers#create` and the Dagger module.